docs/terminal/williamtown.md

```diff
@@ -70,6 +70,22 @@ Voiceless for all aircraft:
 
 All other aircraft coming from ARL CTA will be **Heads-up** Coordinated to WLM TCU.
 
+### Stereo Approaches (Fast Jet Aircraft)
+Fast Jet Aircraft typically arrive back into Willy from Gate 6, at 'F125', as such you can assign them the stereo approach to arrive back into Willy.
+
+!!! Example
+**MAPL45**: "WLM TCU, RTBing via G6 for RWY12"
```

Comment:
This one is radio telephony, so the callsign is used rather than the code
```suggestion
**MAPL45**: "Willy Approach, MAPL45, RTBing via G6 for RWY12"
```